store/views.py

```diff
@@ -77,6 +98,53 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+    '''
+    Check if an instance of the asked book is available.
+    If yes, then set the message to 'success', otherwise 'failure'
+    '''
+    bid = request.POST.get('bid')
+    loan_copy = BookCopy.objects.all()[int(bid)-1]
+    
+    loan_copy.status = True
+    loan_copy.borrow_date = None
+    loan_copy.borrower = None
+    loan_copy.save()
+    response_data['message'] = "success"
+    
+    return JsonResponse(response_data)
 
+@csrf_exempt
+@login_required
+def rateBookView(request):
+    response_data = {
+        'message': None,
+    }
+    '''
+    Check if an instance of the asked book is available.
+    If yes, then set the message to 'success', otherwise 'failure'
+    '''
+    bid = request.POST.get('bid')
+    rating = request.POST.get('rating')
+    print(rating)
 
+    book = Book.objects.filter(id=bid).first()
+    user = request.user
+    rating_data = RatingSystem.objects.filter(book=book,rated_by=user).first()
+    if(rating_data):
+        rating_data.rating = rating
+        rating_data.save()
+    else:
+        rate = RatingSystem.objects.create(book=book, rated_by=user, rating=rating)
+
+    totalrating = 0
+    for ratings_multi in RatingSystem.objects.filter(book=book):
+        totalrating += ratings_multi.rating
+    book.rating = totalrating/RatingSystem.objects.filter(book=book).count()
```

Comment:
While this is correct a more clean and Django-ish way for doing this could be:
```python
RatingSystem.objects.filter(book__pk=book_id).aggregate(Avg('rating'))['rating__avg']
```